test/itemlist_formaction.cpp

```diff
@@ -0,0 +1,154 @@
+#include "catch.hpp"
+
+#include "cache.h"
+#include "itemlist_formaction.h"
+#include "keymap.h"
+#include "stfl/itemlist.h"
+
+#include <unistd.h>
+
+using namespace newsbeuter;
+
+TEST_CASE("itemlist_formaction::process_op()") {
+
+	controller c;
+	newsbeuter::view v(&c);
+	std::string test_browser_file = "/tmp/test_browser.txt";
+	std::string test_url = "http://test_url";
+	std::string cache_path = "test-cache.db";
```

Comment:
Sorry, it's only now that I realized that you don't even need a file—you can just use in-memory DB by passing `":memory:"` instead of the path. (You'll still need `TempFile` for `test_browser_file` though.)